es6-destructuring/main.js

```diff
@@ -0,0 +1,52 @@
+const book1 = {
+  title: 'Goodnight Punpun',
+  author: 'Inio Asano',
+  libraryID: 3353,
+};
+
+const { title, author, libraryID } = book1;
+console.log(
+  `The title of the book is ${title}, the author is ${author}, and the library id is ${libraryID}.`
+);
+
+const book2 = {
+  title: 'Les Fleurs du mal',
+  author: 'Charles Baudelaire',
+  libraryID: 2345,
+};
+
+console.log(
+  `The title of the book is ${book2.title}, the author is ${book2.author}, and the library id is ${book2.libraryID}.`
```

Comment:
Ue destructuring to get these out and assign them to variables with a different name than the field name.